sparse/coo/core.py

```diff
@@ -926,10 +927,9 @@ def transpose(self, axes=None):
             axes = list(reversed(range(self.ndim)))
 
         # Normalize all axe indices to posivite values
-        axes = np.array(axes)
-        axes[axes < 0] += self.ndim
+        axes = _normalize_axis(axes, self.ndim)
 
-        if np.any(axes >= self.ndim) or np.any(axes < 0):
+        if any(a >= self.ndim for a in axes) or any(a < 0 for a in axes):
```

Comment:
This is better placed in `_normalize_axis`.

Reply:
Actually, it seems the function already has this check, so this can be safely removed.